lasagne/layers/merge.py

```diff
@@ -68,22 +70,57 @@ class ElemwiseSumLayer(MergeLayer):
     before a :class:`DenseLayer`, insert separate :class:`DenseLayer` instances
     of the same number of output units and add them up afterwards. (This avoids
     the copy operations in concatenation, but splits up the dot product.)
```

Comment:
> Sometimes you multiply inputs to get multiplicative interaction.

Sure, but then it cannot serve as a replacement for a `ConcatLayer`, can it? I'm maintaining my position that the `Notes` section only applies to the `ElemwiseSumLayer` :) Thanks for moving it!